resources/windows/wix/properties.wxs.erb

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="windows-1252"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Fragment>
+
+    <ComponentGroup Id="Include_Properties" />
+
+    <!-- Add/Remove Programs icon and help link -->
+    <Icon Id="$(var.OurCompanyNameWord).ico" SourceFile="conf\windows\stage\misc\puppetlabs.ico"/>
+    <Property Id="ARPPRODUCTICON" Value="$(var.OurCompanyNameWord).ico" />
+    <Property Id="ARPHELPLINK" Value="$(var.HelpLink)" />
+    <!-- Save destination folder in Add/Remove programs (ARP) registry key -->
+    <SetProperty Id="ARPINSTALLLOCATION" Value="[INSTALLDIR]" After="CostFinalize" />
+
+    <!-- This property is used in the localization strings for the GUI -->
+    <Property Id="VersionUIString" Value="$(var.VersionUIString)" />
```

Comment:
https://github.com/puppetlabs/puppet-agent/blob/feature/master/windows-vanagon-development/configs/projects/puppet-agent.rb#L6-L29 is the source for a lot of these. If this is an erb template, we should be using ruby to populate this stuff